README.md

```diff
@@ -14,3 +14,73 @@ Currently this supports:
 * major releases only
 * all probes registered in separate files (histograms, scalars, events)
 * some select environment data points (more to come)
+
```

Comment:
Maybe a `## Development` header?

Reply:
I left the CRA readme though maybe I will just link to it. It's a little verbose for a small project like this.